application/controllers/HealthController.php

```diff
@@ -99,8 +99,11 @@ public function indexAction()
             ToggleInstanceFeatureCommand::FEATURE_PERFORMANCE_DATA =>
                 $instance->icinga2_performance_data_enabled
         ]);
+
```

Comment:
Why that empty line here now? I keep seeing that you do this, is this a personal preference or do you think it's required?

Reply:
I can remove it directly if it doesn't suit you. I just wanted it to be a bit clearer. I  was advised to add a blank line after a variable declaration and before calling a function.

Reply:
> I was advised to add a blank line after a variable declaration and before calling a function.

There's no rule in the PSR12 styleguide for this. If at all this is personal preference and you may do that if you like, but then only on code you've written personally. Not on code you don't need to touch.

I personally don't add an empty line after variable declarations that are primarily used in the control structure following it. I also write instructions to initialize objects without any empty line in between. The basic principle I follow here is that every part of the code is related to a group. A perfect example are the actions in controllers: query initialization, control initialization, searchbar handling, final query adjustments, exporting, control registration, content preparation, content registration, response handling.